### PR TITLE
Hindi wiki initial pull request

### PR DESCRIPTION
**Pull request by nikhil-11**
Creating pull request to create feature list file for Hindi wiki and making necessary changes in Makefile.
Tried calculating the pop-rate but that did not work on my machine so currently left it random.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1923,9 +1923,46 @@ datasets/hiwiki.autolabeled_revisions.10k_2020.json: \
 		--revert-radius=5 \
 		--verbose > $@
 
-hiwiki_models:
+uning_reports/hiwiki.reverted.md: \
```
Comment (halfak) [inline Makefile]: missing a t

File: Makefile [hunk 2]
```diff
@@ -1923,9 +1923,46 @@ datasets/hiwiki.autolabeled_revisions.10k_2020.json: \
...
+		reverted_for_damage \
+		$(reverted_tuning_statistic) \
+		--label-weight $(reverted_weight) \
+		--pop-rate "true=0.021554310862" \
+		--pop-rate "false=0.978445689138" \
+		--center --scale \
+		--cv-timeout 60 \
+		--debug > $@
+
+models/hiwiki.reverted.gradient_boosting.model: \
```
Comment (halfak) [inline Makefile]: Do the "damaging" and "goodfaith" models instead of "reverted"

File: Makefile [hunk 3]
```diff
@@ -1910,10 +1910,14 @@ hewiki_tuning_reports: \
...
 ############################# Hindi Wikipedia ################################
+datasets/hiwiki.human_labeled_revisions.5k_2020.json:
+	./utility fetch_labels \
+		https://labels.wmflabs.org/campaigns/hiwiki/94/ > $@
+ 
 # From https://quarry.wmflabs.org/query/44835
 datasets/hiwiki.sampled_revisions.10k_2020.json:
-	wget -qO- https://quarry.wmflabs.org/run/467016/output/0/json-lines > $@
-
+   	wget -qO- https://quarry.wmflabs.org/run/467016/output/0/json-lines > $@
```
Comment (halfak) [inline Makefile:1919]: Should just be a TAB

**Comment by halfak** [comment]
This Looks good to me.  It should probably be squash-merged.

**Review by accraze — APPROVED** [review]
:eyes: :+1:

**Pull request merged**